src/Markdig/Helpers/LegacyHelper.cs

```diff
@@ -0,0 +1,20 @@
+#if NET35
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace Markdig.Helpers
+{
+    internal static class LegacyHelper
+    {
+        [MethodImpl(MethodImplOptionPortable.AggressiveInlining)]
+        public static bool HasFlag<TEnum>(this TEnum enumeration, TEnum flags) where TEnum : Enum
+        {
+            Debug.Assert(enumeration.GetType() == flags.GetType());
+            ulong enumVal = Convert.ToUInt64(enumeration);

```

Comment:
actually, looking at the code behind, Convert.ToUInt64 typically incurs an allocation (cast IConvertible) which is just super bad. I would prefer that we remove this method entirely. It's not that annoying in the codebase.

Reply:
Afaik there's no way to see how many people are using the NuGet by framework version right?

Reply:
no unfortunately